### PR TITLE
add test for group deletion

### PR DESCRIPTION
**Pull request by SimoneDutto**
## Description
we were missing a test in the dbmodel pkg for deleted groups.

## Engineering checklist

- [ ] Documentation updated
- [ ] Covered by unit tests
- [ ] Covered by integration tests

### CODE REVIEW

File: internal/dbmodel/group_test.go [hunk 1]
```diff
@@ -36,3 +36,19 @@ func TestGroupEntry(t *testing.T) {
 	c.Assert(result.Error, qt.IsNil)
 	c.Assert(ge3, qt.DeepEquals, ge)
 }
+
+func TestDeleteGroupEntry(t *testing.T) {
```
Comment (ale8k) [inline internal/dbmodel/group_test.go]: TestHardDeleteGroupEntry is a better name. Maybe even a comment too explaining why we're doing this test would be good.

**Review by ale8k — APPROVED** [review]
approving but please do my comment.

File: internal/dbmodel/group_test.go [hunk 2]
```diff
@@ -36,3 +36,20 @@ func TestGroupEntry(t *testing.T) {
 	c.Assert(result.Error, qt.IsNil)
 	c.Assert(ge3, qt.DeepEquals, ge)
 }
+
+// test we hard delete groups
+func TestHardDeleteGroupEntry(t *testing.T) {
```
Comment (alesstimec) [inline internal/dbmodel/group_test.go:41]: i think this test would've passed with soft deletes as well.. what you need to do is
1. create a group
2. delete the created group
3. try to create another group with the same name

**Review by alesstimec — CHANGES_REQUESTED** [review]

File: internal/dbmodel/group_test.go (hunk 2 above)
Comment (SimoneDutto) [reply internal/dbmodel/group_test.go:41]: did it

File: internal/dbmodel/group_test.go [hunk 3]
```diff
@@ -36,3 +36,25 @@ func TestGroupEntry(t *testing.T) {
 	c.Assert(result.Error, qt.IsNil)
 	c.Assert(ge3, qt.DeepEquals, ge)
 }
+
+// test we hard delete groups, so when we can recreate the group with the same name after deleting it.
```
Comment (ale8k) [inline internal/dbmodel/group_test.go]: Godocs should start with the func name

**Review by alesstimec — APPROVED** [review]

**Pull request merged**